.github/workflows/ci_cargo.yml

```diff
@@ -74,16 +71,16 @@ jobs:
           path: /home/runner/work/esp-idf-template/esp-idf-template/github-esp-idf-template
       - name: Generate v5.2
         if: matrix.esp-idf.version == 'v5.2' || matrix.esp-idf.version == 'v5.1'
-        run: cargo generate --path /home/runner/work/esp-idf-template/esp-idf-template/github-esp-idf-template cargo --name ${{ matrix.target }}-${{ matrix.std-config.std }}  --vcs none --silent -d mcu=${{ matrix.target }} -d advanced=true -d espidfver=${{ matrix.esp-idf.version }} -d std=${{ matrix.std-config.std }} -d devcontainer=false -d wokwi=false -d ci=false
+        run: cargo generate --path /home/runner/work/esp-idf-template/esp-idf-template/github-esp-idf-template cargo --name ${{ matrix.target }} --vcs none --silent -d mcu=${{ matrix.target }} -d espidfver=${{ matrix.esp-idf.version }} -d devcontainer=false -d wokwi=false -d ci=false
```

Comment:
I think the removal of `-d advanced=true` is causing the CI failures for the cargo version. You only removed it for cmake.

Reply:
Yes, indeed! Thank you for the hint, @SergioGasquez !